cmd/nitro/nitro.go

```diff
@@ -333,7 +333,7 @@ func mainImpl() int {
 			flag.Usage()
 			log.Crit("validator have the L1 reader enabled")
 		}
-		if !nodeConfig.Node.Staker.Dangerous.WithoutBlockValidator {
+		if nodeConfig.Node.Staker.Strategy != "Watchtower" && !nodeConfig.Node.Staker.Dangerous.WithoutBlockValidator {
```

Comment:
I think this config field is case insensitive, so we should do a case insensitive comparison here (`strings.EqualFold` IIRC, it has a weird name in go). We could also consider calling `stakerStrategyFromString` and comparing it to the watchtower constant; I think either is good.